sdk/eventhub/azure-eventhubs/azure/eventhub/consumer.py

```diff
@@ -114,7 +115,7 @@ def __next__(self):
                 self._offset = EventPosition(event_data.offset, inclusive=False)
                 retried_times = 0
                 if self._track_last_enqueued_event_properties:
-                    self._last_enqueued_event_properties = event_data._runtime_info  # pylint:disable=protected-access
+                    self._last_enqueued_event_properties = event_data._get_last_enqueued_event_properties()  # pylint:disable=protected-access
```

Comment:
This will mean loading the properties for every message if the runtime metric is enabled?

Reply:
right, only when the `_track_last_enqueued_event_properties` is set True.